src/OSPSuite.Core/Domain/Services/MoleculeStartValuesCreator.cs

```diff
@@ -86,7 +86,7 @@ public IMoleculeStartValue CreateMoleculeStartValue(IObjectPath containerPath, s
             Name = moleculeName,
             Dimension = dimension,
             DisplayUnit = displayUnit ?? dimension.DefaultUnit,
-            NegativeValuesAllowed = false
+            NegativeValuesAllowed = false,
          };
```

Comment:
interesting change :-)

Reply:
lol yeah. THat's what happens when you move stuff around :) Not intentional but does not hurt either